scripts/minIO_import.py

```diff
@@ -0,0 +1,132 @@
+import concurrent.futures
+import os
+from pathlib import Path
+
+import boto3
+from tqdm import tqdm
```

Comment:
Can you add a requirements.txt with pinned versions? Pipenv would be better but not necessary

Reply:
👍

Reply:
:+1: